### DISCUSSION TITLE
Usage of .env variables in a .npmrc file

### DISCUSSION THREAD

**Discussion by samuelgoddard** [vercel/next.js#36007] [Help]
Hey, is it possible to use .env variables in a .npmrc file with Next? I've tried the following with no luck

```
**.npmrc
//npm.greensock.com/:_authToken=GSAP_TOKEN
@gsap:registry=https://npm.greensock.comz
```

```
**.env.development
GSAP_TOKEN=tokenhere
```

**Comment by josem-gp** [comment]
@samuelgoddard  I think you should do it like this:
```
**.npmrc
//npm.greensock.com/:_authToken=$GSAP_TOKEN
@gsap:registry=https://npm.greensock.comz
```
By using `$GSAP_TOKEN` you will be able to call the token from the `.env` file. But that alone is not enough. 

You will need to install this `next-env` [package](https://www.npmjs.com/package/next-env) and follow the `Simple Usage`. It worked for me. Good luck!
